contracts/interfaces/draft-IERC7674.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+/**
+ * @dev Temporary Approval Extension for ERC-20 (https://github.com/ethereum/ERCs/pull/358[ERC-7674])
+ *
+ * WARNING: This ERC is not final, and is likely to evolve.
```

Comment:
I think this is implicit from [our docs](https://docs.openzeppelin.com/contracts/5.x/backwards-compatibility#draft_or_pre_final_ercs), and we haven't warned in other interfaces such as `draft-IERC1822.sol` or `draft-IERC6093`.

```suggestion
```